lib/node-detector.js

```diff
@@ -37,6 +40,24 @@ class NodeDetector {
     }
   }
 
+  static async retrieveUsingWhereCommand () {
```

Comment:
This is essentially the same as `retrieveUsingWhichCommand`, so repeating it is not particularly good. It would be better to change the name of a single method to something like `retrieveUsingSystemCall` and then inside, check if this is windows using `appium-support`:

```js
import { system } from 'appium-support';

let cmd = 'which';
if (system.isWindows()) {
  cmd = 'where';
}
stdout = (await exec(cmd, ['node'])).stdout;
```